bundles/org.openhab.core/src/main/resources/OH-INF/config/addons.xml

```diff
@@ -29,6 +29,12 @@
 			<description>Use mDNS network scan to suggest add-ons. Enabling/disabling may take up to 1 minute.</description>
 			<default>true</default>
 		</parameter>
+		<parameter name="suggestionFinderIp" type="boolean">
+			<advanced>true</advanced>
+			<label>IP-based Suggestion Finder</label>
+			<description>Use IP network scan to suggest add-ons.</description>
```

Comment:
Maybe replace 'scan' by 'discovery broadcasts' to make the wording less aggressive.

Reply:
yes, I can change this later